radio/src/targets/common/arm/stm32/sticks_pwm_driver.cpp

```diff
@@ -101,10 +101,10 @@ extern "C" void PWM_IRQHandler(void)
   for (int i=0; i<4; i++) {
     if (PWM_TIMER->SR & (TIM_DIER_CC1IE << i)) {
       uint32_t capture = TIM_GetCapture(i);
+      pwm_interrupt_count++;
```

Comment:
overflow!

Reply:
unless, well ... we only use this variable at startup, it's ok then!
perhaps just add a comment?

Reply:
Was already the case before :)

Reply:
oh yes, true!
my bad ... I add the comment myself